vite.config.ts.timestamp-1721428823603-752cbcb748d68.mjs

```diff
@@ -0,0 +1,20 @@
+// vite.config.ts
+import { vitePlugin as remix } from "file:///C:/Users/temil/Desktop/Temi's%20projects/HNG/hng_boilerplate_remix/node_modules/@remix-run/dev/dist/index.js";
+import { defineConfig } from "file:///C:/Users/temil/Desktop/Temi's%20projects/HNG/hng_boilerplate_remix/node_modules/vite/dist/node/index.js";
+import tsconfigPaths from "file:///C:/Users/temil/Desktop/Temi's%20projects/HNG/hng_boilerplate_remix/node_modules/vite-tsconfig-paths/dist/index.mjs";
```

Comment:
What is going on????

Reply:
will resolve it, am so sorry. my laptop has been switched off